src/fabric_cli/commands/fs/mkdir/fab_fs_mkdir_managedprivateendpoint.py

```diff
@@ -86,16 +87,19 @@ def exec(managed_private_endpoint: VirtualItem, args: Namespace) -> None:
                     # Wait exponentially
                     time.sleep(2**iteration)
                     iteration += 1
-            except Exception:
-                state = "Failed"
+            except Exception as exc:
+                if exc.status_code =='Forbidden' and exc.message == ErrorMessages.Common.forbidden():
```

Comment:
we might change the forbidden() text message in the future which could lead to wrong behavior with this change and state = Failed. do we have anything else we can rely on in the exc? the kind of exception, error codes, anything?